src/libs/Navigation/linkingConfig/getAdaptedStateFromPath.ts

```diff
@@ -165,11 +166,6 @@ function getAdaptedState(state: PartialState<NavigationState<RootStackParamList>
     const featureTrainingModalNavigator = state.routes.find((route) => route.name === NAVIGATORS.FEATURE_TRANING_MODAL_NAVIGATOR);
     const reportAttachmentsScreen = state.routes.find((route) => route.name === SCREENS.REPORT_ATTACHMENTS);
 
-    if (isNarrowLayout) {
-        metainfo.isFullScreenNavigatorMandatory = false;
-        metainfo.isCentralPaneAndBottomTabMandatory = false;
-    }
```

Comment:
Please make sure that this removal doesn't cause any regressions. Find out why this was added at the beginning

Reply:
It was added in [this PR](https://github.com/Expensify/App/pull/38977). The test plan covers this issue (point 6). As far as I remember, I've added the priority mode screen to the `RHP_SCREENS_OPENED_FROM_LHN` to keep it working.